Trappist/src/Promact.Trappist.Web/wwwroot/css/style.scss

```diff
@@ -923,6 +923,39 @@ footer {
     }
 }
 
+.offline-container {
+    position: fixed;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    z-index: 9999;
+    background-color: $color_white;
+
+    .offline-content {
+        position: absolute;
+        top: 50%;
+        left: 0px;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unit for this zero length. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://server12:9005/coding_rules#rule_key=scss%3Azero-units)